### PR TITLE
XW-5117 | New styling for Featured Video on scroll

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/x-wing

### CODE REVIEW

File: app/styles/component/_article-featured-video.scss [hunk 1]
```diff
@@ -16,6 +16,52 @@
 		.jw-related-close {
 			display: none;
 		}
+
+		.jw-controlbar {
+			opacity: 1;
```
Comment (latata) [inline app/styles/component/_article-featured-video.scss]: I think, we don't need it.

File: app/styles/component/_article-featured-video.scss [hunk 2]
```diff
@@ -16,6 +16,52 @@
...
+
+			.jw-controlbar {
+				opacity: 0;
+				pointer-events: none;
+			}
+		}
+
+		.jw-state-paused ~ .article-featured-video__details-wrapper {
+			background: transparent;
+			max-height: 0;
```
Comment (latata) [inline app/styles/component/_article-featured-video.scss]: remove `max-height`

File: app/styles/component/_article-featured-video.scss [hunk 3]
```diff
@@ -16,6 +16,52 @@
...
+			}
+		}
+
+		.jw-state-paused .jw-controls {
+			background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 1));
+		}
+
+		&__details-wrapper {
+			background: $wds-color-white;
+			max-height: 200px;
```
Comment (latata) [inline app/styles/component/_article-featured-video.scss]: remove `max-height`

File: app/styles/component/_article-featured-video.scss [hunk 4]
```diff
@@ -117,20 +168,78 @@
 		display: none;
 	}
 
+	&.is-on-scroll-active &__title {
```
Comment (latata) [inline app/styles/component/_article-featured-video.scss]: remove `&.is-on-scroll-active`

**Review by latata — APPROVED** [review]

**Review by latata — CHANGES_REQUESTED** [review]
Approve missclick ;)

File: app/styles/component/_article-featured-video.scss [hunk 5]
```diff
@@ -117,20 +168,78 @@
...
+		line-height: 1;
+		margin: 0;
+	}
+
+	&.is-on-scroll-active &__time {
+		@extend %wds-font-size-xs;
+		@extend %wds-font-weight-medium;
+
+		background: #272727;
```
Comment (rybmat) [inline app/styles/component/_article-featured-video.scss]: what is that color? maybe it should be some standard wds color with opacity?

File: app/templates/components/article-featured-video.hbs [hunk 6]
```diff
@@ -10,12 +10,11 @@
 		{{article-featured-video-close
 			onTap=(action 'dismissPlayer')
 		}}
-		<div class="article-featured-video__details">
-			<div class="article-featured-video__label wds-font-size-xxs wds-font-weight-bold wds-text-transform-uppercase">
-				{{i18n 'featured-video.watch'}}
+		<div class="article-featured-video__details-wrapper">
+			<div class="article-featured-video__details">
+				<h1 class="article-featured-video__title wds-font-size-s wds-font-weight-bold">{{currentVideoDetails.title}}</h1>
```
Comment (rybmat) [inline app/templates/components/article-featured-video.hbs]: is it ok to use h1 here? shouldn't it be h2? just asking, no strong opinion

**Review by rybmat — APPROVED** [review]

File: app/styles/component/_article-featured-video.scss [hunk 7]
```diff
@@ -117,20 +166,78 @@
...
+	&__title {
+		line-height: 1;
+		margin: 0;
+	}
+
+	&__time {
+		@extend %wds-font-size-xs;
+		@extend %wds-font-weight-medium;
+
+		background: #272727;
```
Comment (latata) [inline app/styles/component/_article-featured-video.scss:178]: If we don't have this color in DS let's ask Derek about it.

File: app/styles/component/_article-featured-video.scss [hunk 8]
```diff
@@ -16,6 +16,50 @@
...
+				width: 60px;
+			}
+
+			.jw-controlbar {
+				opacity: 0;
+				pointer-events: none;
+			}
+		}
+
+		.jw-state-paused ~ .article-featured-video__details-wrapper {
```
Comment (latata) [inline app/styles/component/_article-featured-video.scss:42]: you can put these styles inside `.jw-state-paused {`

File: app/styles/component/_article-featured-video.scss [hunk 9]
```diff
@@ -16,6 +16,50 @@
...
+				transform: translate(0, -100%);
+				transition: transform .3s;
+			}
+
+			.article-featured-video__title {
+				color: $wds-color-white;
+			}
+		}
+
+		.jw-state-paused .jw-controls {
```
Comment (latata) [inline app/styles/component/_article-featured-video.scss:56]: you can put these styles inside `.jw-state-paused {`

**Pull request merged**